### ISSUE TITLE
Mark not restored on compound floating window

### ISSUE THREAD

**Issue by eater** [JonnyHaystack/i3-resurrect#98]
**Describe the bug**
I created a lovely floating + tiled window and I saved it using i3-resurrect. The mark on the outer frame is present in the json file ([attached](https://github.com/JonnyHaystack/i3-resurrect/files/5698445/scratch_layout.json.txt)), but it is not restored when I restore the window.

**To Reproduce**
1. Create a floating window containing multiple tiles.
2. Mark its outermost parent, so that you can send it to the scratchpad.
3. `i3-resurrect save -w __i3_scratch -p bunchawindows`
4. Close float.
5. 3. `i3-resurrect restore -p bunchawindows`

**Expected behavior**
The restored window should have the mark, but it doesn't.

**System information (please complete the following information):**
 - Linux distribution: Ubuntu 18.04
 - i3-resurrect version (output of `i3-resurrect --version`): 1.4.3
 - i3 version (output of `i3 -v`): 4.19
 - Python version (output of `python -V`): 3.6.9

**Comment by JonnyHaystack** [comment]
How are you even able to do `i3-resurrect save -w __i3_scratch -p bunchawindows`? According to #97 I would've expected this not to even save successfully.

**Comment by JonnyHaystack** [comment]
Can you send the saved layout.json? Does it contain all the marks that it's supposed to?

**Comment by eater** [comment]
Yeah, I attached the layout.json to the original issue. The mark is called "msgs".

**Comment by JonnyHaystack** [comment]
Oh sorry I must be blind, didn't reread properly before asking that.. Hmm I'm not sure there's anything that I can do if the mark is saved in the layout file but not being restored by i3 itself. That would probably need changing in i3 itself.
